src/routes/account.js

```diff
@@ -201,9 +201,9 @@ router.get('/connect/discord', requireLoginMiddleware, async (request, response)
 		tokens = await discordOAuth.tokenRequest({
 			code: request.query.code,
 			scope: 'identify guilds',
-			grantType: 'authorization_code',
+			grantType: 'authorization_code'
 		});
-	} catch (error) {
+	} catch (ignored) {
```

Comment:
What is this doing? I don't see `ignored` being used anywhere, I assume the name was changed to make it clear that the error isn't being used?

If that's the case, why not just remove it entirely? The `catch` block does not require the error to be captured:

```suggestion
	} catch {
```

Reply:
In JavaScript the catch parameters are required, I believe I tested this and removing it works in TS but not in pure JS

I can test again to be sure

Reply:
Interesting - I must have been receiving the error before I set the ECMA version to latest - ig this is probably newish syntax

Reply:
According to MDN the parameter was made optional after the initial spec for `try-catch`, yes. Most browsers implemented it in 2018, with the exception of Android's WebView which had it mid 2017 and then some which waited until 2020

So definitely recent-ish syntax. Idk when the spec was updated though